### PR TITLE
docs: fix typo, trailing space

### PR DESCRIPTION
**Pull request by tmummert**
Fixed:
`runtime/doc/channel.txt` -> typo
`runtime/doc/ft_rust.txt` -> trailing space

### CODE REVIEW

File: runtime/doc/ft_rust.txt [hunk 1]
```diff
@@ -276,7 +276,7 @@ g:rust_keep_autopairs_default ~
 
 	Don't override auto-pairs default for the Rust filetype. The default
 	is 0.
- 
+
```
Comment (zeertzjq) [inline runtime/doc/ft_rust.txt:279]: I think this blank line should be removed.

Comment (tmummert) [reply runtime/doc/ft_rust.txt:279]: There are some irregularities regarding separators:

```
<empty line>
<empty line>
<separator>
```

vs.
```
<empty line>
<separator>
```

I will adjust everything to the latter.

At the same time, two blank lines should be kept between each setting for better readability.

Comment (zeertzjq) [reply runtime/doc/ft_rust.txt:279]: I'm specifically talking about this one. Please don't change other places.

**Comment by dundargoc** [comment]
Thank you for your contribution! We collect all typo fixes and similar small fixes into a single pull request and merge it once it gets big enough: https://github.com/neovim/neovim/pull/25585

**Pull request closed without merging**